src/ml-service/packages/sensor_location/locate/app.py

```diff
@@ -0,0 +1,111 @@
+from flask import Flask, jsonify, request
+from flask_pymongo import PyMongo
+from modules import locate_model, locate_helper 
+#from flask_cors import CORS, crossorigin
+from flask_cors import CORS
+import os
+import sys
+import ast
+import json
+ 
+app = Flask(__name__)
+CORS(app)
+
+# add mongo url to flask config, so that flask_pymongo can use it to make connection
+app.config["MONGO_URI"] = os.getenv('MONGO_URI')
+mongo = PyMongo(app)
+
+@app.route('/', methods=['GET', 'POST'])
+def main():
+    return 'ok'
+
+@app.route('/api/v1/parishes', methods=['GET', 'POST'])
+def place_sensors():
+    '''
```

Comment:
Great work, could we move some of this business logic to the `controllers` directory and then define the end points under `routes` directory? Just like in our Sequence Diagrams.

Reply:
I think this file could either be renamed to `route.py` OR moved to` locate/routes/api.py` where the business logic (functions) are imported from a `controllers` file.